COPYRIGHT

```diff
@@ -398,43 +372,16 @@ SOFTWARE.
 lua-resty-acme
 
 https://github.com/fffonion/lua-resty-acme
-https://github.com/fffonion/lua-resty-acme#copyright-and-license
```

Comment:
It seems like the newly generated copyright is missing some license, for example this one. cc @fffonion since you're the author of this lib

Reply:
Regenerated the COPYRIGHT and it's back.